Add May 2024 newsletter (#2141)
writethedocs/www: docs/blog/newsletter-may-2024.rst

:og:image: https://www.writethedocs.org/_static/logo-opengraph.png

.. post:: May 06, 2024
    :tags: newsletter

#########################################
Write the Docs Newsletter – May 2024
#########################################

Hello once again, fellow documentarians. I hope all is well with you and yours.

Our online Atlantic conference is planned for September and there's still time to get in your proposal to speak. Check out the `Call for Proposals </conf/atlantic/2024/cfp/>`__ and get your ideas in by May 15. The Atlantic conference welcomes speakers from around the world and you don't even have to travel to give the talk. So if you've ever considered giving a talk before, send in your proposal and give it a try.

If you'd like some inspiration from past speakers, the Portland conference wrapped up since our last newsletter. See the `conference recap </conf/portland/2024/news/thanks-recap/>`__ and see `videos of all the talks <https://www.youtube.com/playlist?list=PLZAeFn6dfHpm4FboYSTD1Bs8Yp8k_JvAL>`__ to get a taste of what the conference was like. And then get inspired and `submit a talk for Atlantic 2024 </conf/atlantic/2024/cfp/>`__!

> In other exciting news... the 2023 Documentation Salary Survey results are out! Thanks to a record number of submissions and finer-grained location data, we are able to publish median salaries for many more regions that we have been able to cover previously. And it's possible, we're also showing more data on salaries, such as the 25th and 75th percentile values. For all the numbers, head over to the `2023 salary survey results </surveys/salary-survey/2023/>`__.

Reviewer: ```suggestion
In other exciting news... the 2023 Documentation Salary Survey results are out! Thanks to a record number of submissions and finer-grained location data, we are able to publish median salaries for many more regions than we have been able to cover previously. And it's possible, we're also showing more data on salaries, such as the 25th and 75th percentile values. For all the numbers, head over to the `2023 salary survey results </surveys/salary-survey/2023/>`__.
```

Reviewer: A question about `And it's possible, we're also showing more data on salaries` -- should it be something like "And we're also showing..."? The "it's possible" part threw me b/c it seems like we'd know if we were showing more data -- I thought it might be an artifact from an earlier version or something.

Author: Oh, that's a poor job of editing on my part. It should be `And where it's possible` because it's not true with all of the questions.
```suggestion
In other exciting news... the 2023 Documentation Salary Survey results are out! Thanks to a record number of submissions and finer-grained location data, we are able to publish median salaries for many more regions than we have been able to cover previously. And where it's possible, we're also showing more data on salaries, such as the 25th and 75th percentile values. For all the numbers, head over to the `2023 salary survey results </surveys/salary-survey/2023/>`__.
```

Another significant statistic is how documentation affects product uptake and retention. Tracking user journeys from documentation to product features and comparing documentation consumption with customer retention rates may illustrate its worth in increasing product utilization and customer satisfaction.

However, it is critical to understand the limitations of measurements. While quantitative data might give valuable insights, it may only capture some of the user experience or the underlying causes for user behavior. Any metrics should be combined with qualitative input, such as user interviews and usability testing. You want to know how users are using your docs.

Relying solely on metrics without context or interpretation can lead to misunderstanding or misaligned goals. Metrics should be part of a comprehensive strategy that aligns with company goals and prioritizes delivering an exceptional user experience. It is critical to select metrics carefully, augment quantitative data with qualitative insights, and interpret metrics in light of larger corporate goals and user demands.

-------------------------
Is Docs-as-Code Worth It?
-------------------------

> The busiest topic in the WtD Slack this past month began in the `#docs-as-code channel <https://writethedocs.slack.com/archives/C72NZ18FR>`__ with a question about what bugs people about a docs-as-code approach. It set off a series of conversations touching on issues with docs-as-code as well as what people hope to gain from such an approach and what the next steps in its evolution could be.

Author: ```suggestion
The busiest topic in the WTD Slack this past month began in the `#docs-as-code channel <https://writethedocs.slack.com/archives/C72NZ18FR>`__ with a question about what bugs people about a docs-as-code approach. It set off a series of conversations touching on issues with docs-as-code as well as what people hope to gain from such an approach and what the next steps in its evolution could be.
```

:og:image: https://www.writethedocs.org/_static/logo-opengraph.png

.. post:: May 07, 2024
    :tags: newsletter

#########################################
Write the Docs Newsletter – May 2024
#########################################

Hello once again, fellow documentarians. I hope all is well with you and yours.

> Our online Atlantic 2024 conference is planned for September and there's still time to get in your proposal to speak. Check out the `Call for Proposals </conf/atlantic/2024/cfp/>`__ and get your ideas in by May 15. The Atlantic 2024 conference welcomes speakers from around the world and you don't even have to travel to give the talk. So if you've ever considered giving a talk before, send in your proposal and give it a try.

Reviewer 2: ```suggestion
Our online Atlantic 2024 conference is happening in September and there's still time to get in your proposal to speak. Check out the `Call for Proposals </conf/atlantic/2024/cfp/>`__ and get your ideas in by May 15. The Atlantic 2024 conference welcomes speakers from around the world and you don't even have to travel to give the talk. So if you've ever considered giving a talk before, send in your proposal and give it a try.
```

Reviewer 2: <3 nice

If you'd like some inspiration from past speakers, the Portland 2024 conference wrapped up since our last newsletter. See the `conference recap </conf/portland/2024/news/thanks-recap/>`__ and see `videos of all the talks <https://www.youtube.com/playlist?list=PLZAeFn6dfHpm4FboYSTD1Bs8Yp8k_JvAL>`__ to get a taste of what the conference was like. And then get inspired and `submit a talk for Atlantic 2024 </conf/atlantic/2024/cfp/>`__!

> We'd also like to announce the Australia 2024 conference dates. This year, the conference will take place in Melbourne on 28 and 29 November. We'll share more details around venue, tickets, and the CFP process in a couple of weeks.

Reviewer 2: @swapnilogale @CollierCZ are we nixing this entirely? Or maybe rewording to a more vague "finishing touches etc"

Reviewer 3: Happy to replace " on 28 and 29 November" with "late November". Rest of the text sounds fine.

Author: ```suggestion
We'd also like to announce the Australia 2024 conference dates. This year, the conference will take place in Melbourne in late November. We'll share more details around venue, tickets, and the CFP process in a couple of weeks.
```

Military and aerospace contracts include provisions for support and support costs get minimized with effective documentation. Therefore, documentarians may be seen as necessary team members if support is considered part of the bottom line.

Some people indicated that they, as documentarians, are under Technology, Product, Engineering, Support, or even Marketing VPs and that they have been in effective documentation departments. They don’t see a need for an a documentation VP.

Could it make sense to have one high-level department for both technical documentation and marketing-related content, such as VP of Communications? The benefit of having one department for both may be consistent usage and branding, but in many companies this is achieved by having style guides (and other related job aids) and branding guidelines. But, of course, these are only effective for company-wide consistency when actually used.

---------------------

> Docs Metrics To Track

Reviewer 2: "to track" feels a little weak to me. Just "Docs Metrics"?

> ---------------------
> Docs Metrics To Track
> ---------------------

Author: ```suggestion
---------------------------------
Gaining Insight from Docs Metrics
---------------------------------
```
